framework/packages/abstract-std/src/objects/ownership/cw721.rs

```diff
@@ -0,0 +1,42 @@
+use cosmwasm_schema::cw_serde;
+
+/// Expiration represents a point in time when some event happens.
+/// It can compare with a BlockInfo and will return is_expired() == true
+/// once the condition is hit (and for every block in the future)
+#[cw_serde]
+#[derive(Copy)]
+pub enum Expiration {
+    /// AtHeight will expire when `env.block.height` >= height
+    AtHeight(u64),
+    /// AtTime will expire when `env.block.time` >= time
+    AtTime(cosmwasm_std::Timestamp),
+    /// Never will never expire. Used to express the empty variant
+    Never {},
+}
+
+#[cw_serde]
+pub struct Approval {
+    /// Account that can transfer/send the token
+    pub spender: String,
+    /// When the Approval expires (maybe Expiration::never)
+    pub expires: Expiration,
+}
+
+#[cw_serde]
+pub struct OwnerOfResponse {
+    /// Owner of the token
+    pub owner: String,
+    /// If set this address is approved to transfer/send the token as well
+    pub approvals: Vec<Approval>,
+}
```

Comment:
```suggestion
#[cw_serde]
pub struct OwnerOfResponse {
    /// Owner of the token
    pub owner: String,
}
```
We only care about owner right now, should we skip deserializing approvals?

Reply:
BTW https://github.com/CosmWasm/cosmwasm/blob/3b15bfba6406cac2f807b9af93077c04d27e79c5/CHANGELOG.md?plain=1#L157

Reply:
ahhh nice! Yeah then we can keep it out but leave it in as a commented value. 
```rust
pub struct OwnerOfResponse {
	    /// Owner of the token
	    pub owner: String,
	    /// If set this address is approved to transfer/send the token as well
	    // Left out as an optimization.
	    // pub approvals: Vec<Approval>,
	}
```

Reply:
Sure!